docker/webui/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM opensuse:13.2
+MAINTAINER Jan Sedlak <jsedlak@redhat.com> Josef Skladanka <jskladan@redhat.com>
+
+RUN useradd -m geekotest && \
+    zypper ar -f http://download.opensuse.org/repositories/devel:openQA:stable/openSUSE_13.2/devel:openQA:stable.repo && \
+    zypper ar -f obs://devel:openQA:13.2/openSUSE_13.2 openQA-perl-modules && \
+    zypper ar -f http://download.opensuse.org/repositories/devel:/languages:/perl/openSUSE_13.2 perl-libs && \
+    zypper --non-interactive --gpg-auto-import-keys in openQA ca-certificates
+
+# setup apache
+RUN gensslcert && \
+    a2enmod headers && \
+    a2enmod proxy && \
+    a2enmod proxy_http && \
+    a2enmod proxy_wstunnel && \
+    a2enmod ssl && \
+    a2enflag SSL
+ADD openqa-ssl.conf /etc/apache2/vhosts.d/openqa-ssl.conf
+ADD openqa.conf /etc/apache2/vhosts.d/openqa.conf
+
+# set-up shared data and configuration
+RUN rm -rf /etc/openqa/openqa.ini /etc/openqa/client.conf \
+      /var/lib/openqa/share/factory /var/lib/openqa/share/tests \
+      /var/lib/openqa/db/db.sqlite /var/lib/openqa/testresults && \
```

Comment:
if you create those directories as symlinks before actually installing the packages rpm would just follow the links so you'd not have to recreate everything yourself